ui/src/components/Timeline/Timeline.test.tsx

```diff
@@ -255,7 +255,7 @@ it('allows creating new items', async () => {
   // 'Event' is the default schema
   expect(screen.getByRole('textbox', { name: 'Name' })).toBeInTheDocument();
   expect(screen.getByRole('textbox', { name: 'Start date' }));
-  expect(screen.getByRole('textbox', { name: 'End date' }));
+  expect(screen.getByRole('textbox', { name: 'End date (optional)' }));
```

Comment:
For this, I don't suppose we have the translation for optional for other languages do we?

Reply:
It will use translations (as soon as someone has translated this string in Transifex)

https://github.com/alephdata/aleph/pull/2921/files#diff-391f5ad77c4995b4373c13316eafa3d5b2987c7f6a7ebcb611eabfbde3323469R101-R106